src/components/test-category-icon/test-category-icon.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'test-category-icon',
+  templateUrl: 'test-category-icon.html'
+})
+export class TestCategoryIconComponent {
+  @Input()
+  category: string;
+
+  categoryIcons = {
```

Comment:
Could we use a string [Enum](https://www.typescriptlang.org/docs/handbook/enums.html) for the categories?

Reply:
Done